.github/workflows/examples-efr32.yaml

```diff
@@ -15,10 +15,9 @@
 name: Build example - EFR32
 
 on:
-    push:
-        branches:
-            - master
-            - 'v*-branch'
+  push:
+    branches-ignore:
```

Comment:
I think that you can make changes here as long as you will not change the indentation (keep it as it is right now - 2 spaces).